docs/contribute.md

```diff
@@ -0,0 +1,125 @@
+# Contribute
+
+- [Submitting a Merge Request](#submit-mr)
+- [Commit Message Guidelines](#commit)
+
+## <a name="submit-mr"></a> Submitting a Merge Request (MR)
+
+Before you submit your Merge Request (MR) consider the following guidelines:
+
+To the first point
+
+1. Make your changes in a new git branch:
+
+   > Note! The name of the new branch must have only alphanumeric chars and a hyphen.
+
+   > Be aware! All new branches should be created from `main` branch
+
+   ```shell
+   git checkout -b my-fix-branch main
+   ```
+
+2. Create your bugfix or feature.
+3. Follow our Coding Rules.
+4. Commit your changes using a descriptive commit message that follows our
+   [commit message conventions](#commit). Adherence to these conventions
+   is necessary because release notes are automatically generated from these messages.
+
+   ```shell
+   git commit -a
```

Comment:
Let's discuss this point. As I recently used another appoach where while you are working on a branch you make additional commits for fixes and other stuff, it allows easily access this by reviewer. Instead we can just squash this when we merge data.